scripts/custom/map_marital_status.py

```diff
@@ -18,7 +18,7 @@ class MaritalStatus(Enum):
 
 
 def map_marital_status(code):
-    """Map marital status for mimic database
+    """Map MIMIC marital status
```

Comment:
Is it necessary to precise that this marital status mapping is for MIMIC ? Could be useful for other databases

Reply:
The source codes are very specific to mimic db, but you're right we might use this for future databases, that's why we keep the name of the function generic.